### PR TITLE
[T4A2][T14-B1]Teh Ee Thai

### PR DESCRIPTION
**Pull request by tet54**

### CODE REVIEW

**Comment by tet54** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    private static int nextSequenceNumber;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:18]: from a readability point of view, you can set `nextSequenceNumber` to 1

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -222,7 +222,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareFind(String args) {
+    private static Command prepareFind(String args) {
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:225]: you will need to update the parser test file that uses `parseCommand` method

**Comment by okkhoy** [comment]
@tet54 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**